tests/unit/frost-file-picker-test.js

```diff
@@ -0,0 +1,21 @@
+import {expect} from 'chai'
+import {setupComponentTest} from 'ember-mocha'
+import {helpers} from 'ember-prop-types'
+import {beforeEach, describe, it} from 'mocha'
+const {validatePropTypes} = helpers
+
+describe('Unit / Component / frost-file-picker /', function () {
+  setupComponentTest('frost-file-picker', {unit: true})
+
+  let component
+  beforeEach(function () {
+    component = this.subject({accept: '*'})
+  })
+
+  describe('`proptypes` validation', function () {
+    it('should have valid `proptypes` declaration', function () {
+      validatePropTypes(component)  // raises an exception if it fails
```

Comment:
Couldn't you use `to.not.throw()`?

Reply:
I didn't know I could do that. Thanks for suggesting that.

```js
      expect(function () {
        validatePropTypes(component)
      }).not.to.throw()
```
This seems to also work, and I definitely like it better. Would you prefer that I add that as a separate commit, or squashed with this one?

Reply:
Doesn't matter to me. I guess it would be better as a separate commit so it'll be easier to track your PR fixes.

Reply:
That `.to.throw()` might be a problem... There was a change that makes that not work unless you do this:
```
// Hack to fix test failing when using `throw` - @dafortin 2017.06.21
// Issue introduced by https://github.com/emberjs/ember.js/pull/14898
Test.adapter = Test.MochaAdapter.extend({
  exception (error) {
    throw error
  }
}).create()
```
https://github.com/ciena-frost/ember-frost-core/blob/b023f7c9a0094a8ec72febed36062d4d22b28b34/tests/integration/components/frost-toggle-test.js#L11

Reply:
Hmm, while I'm not entirely caught up with that thread, that seems to indicate issues when running Ember code that utilizes the run loop. I haven't seen this issue when running pure js stuff. Although, I'm not sure `validatePropTypes` has Ember run loop aware code in it.

Reply:
I think it's safe to keep as is. I read that thread thoroughly and it indicates that code inside a run loop will swallow errors unless you configure Mocha to rethrow it. I checked `validatePropTypes` and it isn't using any run loop aware mechanisms except maybe `get` but that's executed in a synchronous fashion and shouldn't be swallowing. Although adding that re-throw wouldn't hurt. @gknoy @juwara0 What do you think?

Reply:
Interestingly, the test doesn't actually execute anyway if its failure condition is met: the ` `beforeEach` will fail with the error message I mention in my PR description before it even executes the `it(...)` clause, because the changes to our `environment.js` ensure that we validate our proptypes, and that happens when the component is initialized.

I had intended the test as a safeguard against changes to the environment + proptypes, but it appears not to do that as written.  I've fixed the test to actually fail if the environment.js isn't configured to force validation, and it fails correctly even without @juwara0 's  suggestion above:
```
AssertionError: expected [Function] to not throw an error but 
 'Error: [<dummy@component:frost-file-picker::ember480>]: Unknown propType undefined' 
  was thrown
```

I can either fix that, or I can remove the test entirely and trust the environment.js. :)

Reply:
For future consideration, when it comes to making decisions such as these that revolve around the testing of the throwing of errors, I would recommend implementing the approach that accounts for the _potential_ of a run loop being used even if currently isn't, as long as the implementation can correctly account for the current state of code and the potential future state.   I'm not advocating trying to be future-proof against all code that has never been written yet but it is not without possibility that the PropTypes code would one day leverage a run loop.  While most test code will then fail if something it is testing has been changed, when it comes to error-related coding and testing it is likely that such a failure would not be captured correctly, rendering the test inaccurate.